SQL/2016-11-04-PreferenceNotifications.sql

```diff
@@ -0,0 +1,40 @@
+DROP TABLE IF EXISTS `notification_methods`;
+CREATE TABLE `notification_methods` (
+	`NotificationMethodID` int(10) NOT NULL,
+    `Method` varchar(255) NOT NULL,
+	PRIMARY KEY (`NotificationMethodID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+INSERT INTO `notification_methods` VALUES (1, "Within Loris");
+INSERT INTO `notification_methods` VALUES (2, "Email");
+INSERT INTO `notification_methods` VALUES (3, "Text Message");
```

Comment:
I don't see why this table is required. You can't add a notification method without altering the code anyways, so there's no reason to use a table instead of an appropriate const somewhere in the code.